csv_export.php

```diff
@@ -68,16 +68,8 @@
 
 $t_filename = csv_get_default_filename();
 
-# Send headers to browser to activate mime loading
```

Comment:
Does this really add value?  I'm not sure this is useful.  I actually don't understand the point it is making.